pkg/custompluginmonitor/custom_plugin_monitor.go

```diff
@@ -41,35 +41,43 @@ type customPluginMonitor struct {
 
 // NewCustomPluginMonitorOrDie create a new customPluginMonitor, panic if error occurs.
 func NewCustomPluginMonitorOrDie(configPath string) types.Monitor {
```

Comment:
We can probably remove this once you redirect the flags above.

Reply:
Done. Thanks!